ocds.ttl

```diff
@@ -21,10 +21,6 @@
 # #################################################################
 # 
 # 
-# http://www.w3.org/2000/01/rdf-schema#commnent
-
-rdfs:commnent a owl:AnnotationProperty .
-# 
 # http://www.w3.org/2000/01/rdf-schema#description
 
 rdfs:description a owl:AnnotationProperty .
```

Comment:
I don't get how the reasoning engine could produce this triple:

- rdfs:description doesn't exist (neither does rdf:description)
- rdfs:description is not used in the resulting file

Basically it doesn't exist and we don't need since we have rdfs:comment.